packages/sdk/src/reads/getIsExternalPositionsValuesZero.ts

```diff
@@ -0,0 +1,33 @@
+import { publicClientMainnet } from "../../tests/globals.js";
+import { getActiveExternalPositions } from "./getActiveExternalPositions.js";
+import { getExternalPositionAssets } from "./getExternalPositionAssets.js";
+import type { Address } from "viem";
+
+export async function getIsExternalPositionsValueZero(vaultProxy: Address) {
+  const addresses = await getActiveExternalPositions(publicClientMainnet, {
+    vaultProxy,
+  });
+
+  const values = await Promise.all(
+    addresses.map(async (address) => {
+      const { debtAssets, managedAssets } = await getExternalPositionAssets(publicClientMainnet, {
```

Comment:
pass client as a parameter, we support different networks, and also publicClientMainnet is only for tests purposes